.config/hypr/scripts/i-screenshot.sh

```
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Define the path to save the screenshot
+ppath=~/Pictures/Screenshots/Screenshot-"$(date +%F_%T)".png
+
+# Take a screenshot of a selected region and copy it to clipboard, then save it to the defined path
+grim -g "$(slurp)" - | wl-copy && wl-paste > "$ppath" | dunstify "Screenshot of the region taken at $(date +%F_%T)" -t 1000
```

Shellcheck (0.10.0)

[error] This redirection overrides the output pipe. Use 'tee' to output to both.

(SC2260)